### PR TITLE
docs: add ES6 Map

### PR DESCRIPTION
**Pull request by kyeotic**
This is my first PR. I want to add more of the types for #139, but I want to make sure I understand what is required first.

### CODE REVIEW

**Comment by RusinovAnton** [comment]
Thank you for this! I'm going to make some remarks later today

File: public/docs/Map/clear.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+## Syntax
+
+```js
+map.clear();
+```
+
+## Usage Examples
+
+```js
+var myMap = new Map();
```
Comment (RusinovAnton) [inline public/docs/Map/clear.md]: Please use modern ES6 syntax, `const` or `let` instead of var

**Comment by kyeotic** [comment]
@RusinovAnton I believe this addresses all the formatting concerns. If this looks good I will do Set next. Your call on whether or not thats a seperate PR

**Comment by RusinovAnton** [comment]
@kyeotic I believe those requested changes are related to this current PR, but if you feel like doing it in a separate one I'm fine with that. Also, I could do that formatting myself, most of the job is done here

**Comment by RusinovAnton** [comment]
I would prefer to push my commits onto yours and then merge, so I'd cherry-pick these into my PR.

**Comment by kyeotic** [comment]
@RusinovAnton I've added the `Set` docs

**Comment by kyeotic** [comment]
@RusinovAnton Do you still need any changes on this?

**Comment by RusinovAnton** [comment]
Yes, but I will be able to work on then not earlier than Saturday, thanks

**Comment by RusinovAnton** [comment]
Sorry for the delay. I've merged your commits within the https://github.com/RusinovAnton/js-tldr/pull/145
Your contribution is much appreciated!

**Comment by kyeotic** [comment]
Awesome, glad I could contribute to this beautiful project :)

**Pull request closed without merging**